src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Subscales/Subscale/Subscale.tsx

```diff
@@ -19,14 +31,20 @@ export const Subscale = ({ items }: SubscaleProps) => {
 
   return (
     <>
-      {items?.map(({ type, id, name, items }) => (
+      {items?.map(({ type, id, name, items, additionalInformation }) => (
         <Fragment key={id}>
           {items?.length ? (
             <Accordion title={name} key={id}>
-              <Subscale items={items} />
+              <>
+                {additionalInformation && renderAdditionalInformation(additionalInformation)}
```

Comment:
can you move `additionalInformation` condition as a part of `renderAdditionalInformation`:
if (!renderAdditionalInformation) return null;